tests/behat/features/prof/03-report/03-fees.feature

```diff
@@ -1,123 +0,0 @@
-#      | scoo-reason-test | scco-reason               |
-#      | 1,690            | grand-total-charged       |
-#      | 9,876            | grand-total-received      |
-#
-#
-#  Scenario: fees edit + remove
-#    Given I save the application status into "prof-fees-expenses-before-edit"
-#    And I am logged in as "behat-prof1@publicguardian.gsi.gov.uk" with password "Abcd1234"
-#    And I click on "pa-report-open" in the "client-01000010" region
-#    And I click on "edit-prof_current_fees"
-#    When I click on "edit-fee" in the "service-fee-annual-report" region
-#    Then the following fields should have the corresponding values:
-#      | prof_service_fee_type_serviceTypeId_0 | annual-report |
-#    And the step with the following values CAN be submitted:
-#      | prof_service_fee_type_serviceTypeId_1 | annual-management-interim |
-#    And the following fields should have the corresponding values:
-#      | prof_service_fee_type_assessedOrFixed_0         | fixed    |
-#      | prof_service_fee_type_amountCharged             | 1,234.00 |
-#      | prof_service_fee_type_paymentReceived_0         | yes      |
-#      | prof_service_fee_type_amountReceived            | 9,876.00 |
-#      | prof_service_fee_type_paymentReceivedDate_day   | 01       |
-#      | prof_service_fee_type_paymentReceivedDate_month | 01       |
-#      | prof_service_fee_type_paymentReceivedDate_year  | 2018     |
-#    And the step with the following values CAN be submitted:
-#      | prof_service_fee_type_assessedOrFixed_1 | assessed |
-#      | prof_service_fee_type_amountCharged     | 11223344 |
-#      | prof_service_fee_type_paymentReceived_0 | no       |
-#    # summary page again fix
-#    Then each text should be present in the corresponding region:
-#      | Yes              | has-fees                               |
-#      | 11,223,344       | assessed-fee-annual-management-interim |
-#      | Yes              | previous-estimates                     |
-#      | scoo-reason-test | scco-reason                            |
-#    # remove appointment
-#    Given I load the application status from "prof-fees-expenses-before-edit"
-#    When I click on "delete" in the "assessed-fee-annual-management-interim" region
-#    Then I should not see the "assessed-fee-annual-management-interim" region
-#    # change initial question to "no", check you are in start page
-#     # change to yes
-#    When I click on "edit-exist" in the "has-fees" region
-#    And the step with the following values CAN be submitted:
-#      | prof_service_fees_currentProfPaymentsReceived_1 | no |
-#    And I should see "No" in the "has-fees" region
-#    And I should not see the "assessed-fee-annual-management-interim" region
-#    And I should not see the "previous-estimates" region
-#    And I should not see the "scco-reason" region
-#    # restore data before scenario
-#    And I load the application status from "prof-fees-expenses-before-edit"
```

Comment:
Why is this commented out? Has it been replaced?

Reply:
Ah i see, it was already commented out and its just been removed is that correct?

Reply:
yes, I've just removed it